docs/conf.py

```diff
@@ -341,20 +147,20 @@ def process_sections(
 
 # -- Options for LaTeX output ------------------------------------------------
 
-latex_elements = {
-    # The paper size ('letterpaper' or 'a4paper').
-    #
-    # 'papersize': 'letterpaper',
-    # The font size ('10pt', '11pt' or '12pt').
-    #
-    # 'pointsize': '10pt',
-    # Additional stuff for the LaTeX preamble.
-    #
-    # 'preamble': '',
-    # Latex figure (float) alignment
-    #
-    # 'figure_align': 'htbp',
-}
+# latex_elements = {
+#     The paper size ('letterpaper' or 'a4paper').
+#
+#     'papersize': 'letterpaper',
+#     The font size ('10pt', '11pt' or '12pt').
+#
+#     'pointsize': '10pt',
+#     Additional stuff for the LaTeX preamble.
+#
+#     'preamble': '',
+#     Latex figure (float) alignment
+#
+#     'figure_align': 'htbp',
+# }
```

Comment:
Can we delete this?

Reply:
Sure, this might be useful for building PDF documentation but I have no experience with that so that will have to wait for another day.